lib/loader/index.js

```diff
@@ -98,9 +99,10 @@ function postInstantiate(extendedExports, instance) {
   /** Gets the runtime type info for the given id. */
   function getInfo(id) {
     const U32 = new Uint32Array(memory.buffer);
-    const count = U32[__rtti_base >>> 2];
+    const rtBase = rttiBase();
+    const count = U32[rtBase >>> 2];
```

Comment:
Could use `rttiBase()` on this line to perform a check once, and `__rtti_base` below